services/llm/anthropic/service.go

```diff
@@ -67,7 +67,7 @@ func (s *service) Response(ctx context.Context, instructions, input string, maxT
 		MaxTokens:   2500,
 	})
 	if err != nil {
-		return nil, s.error(fmt.Errorf("error calling Anthropic API: %w", err), instructions, input)
```

Comment:
these errors are unwound in the web layer to find the service error so not a lot of point in wrapping here